build.gradle

```diff
@@ -751,20 +753,20 @@ project(':clients') {
   }
 
   task determineCommitId {
-    ext.commitId = "unknown"
```

Comment:
It seems like we lost the behavior of using "unknown" the relevant info from git cannot be found. This would be the case if you were building from one of the source archives from a release. I'm not sure what would happen downstream if you ended up with `null` here.